apps/riven/.env.test

```
@@ -10,10 +10,10 @@ RIVEN_SETTING__vfsDebugLogging=false
 RIVEN_SETTING__vfsMountPath="/mnt/test-vfs-mount"
 
 ## Clear queues on startup
-RIVEN_SETTING__unsafeClearQueuesOnStartup=false
+RIVEN_SETTING__unsafeWipeRedisOnStartup=false
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The RIVEN_SETTING__unsafeWipeRedisOnStartup key should be in uppercase

(LowercaseKey)